MovingBlocks/Terasology
# Fix: Reflect sky in water

### Contains

Sky was not being reflected in the water:
![Terasology-190403002354-1366x706](https://user-images.githubusercontent.com/22580625/55663966-34b9c900-57ec-11e9-9db7-d8e2533a0fce.png)

This is because the sky sphere was not being culled from the correct side. The sky is now culled from the correct side in the relevant reflection node.

### How to test

- Run the game with global reflective water
- See some sky reflections in the water

### Outstanding before merging

None
